editor/src/components/canvas/ui-jsx-canvas-renderer/scene-root.tsx

```diff
@@ -238,6 +242,7 @@ export const SceneRootRenderer = betterReactMemo(
       ...sceneProps.props,
       [UTOPIA_SCENE_PATH]: scenePath,
       [UTOPIA_UIDS_KEY]: sceneUID,
```

Comment:
Can we bin off the uid key with this change?

Reply:
🤔 actually I think we could. however, since the data-uid is inserted as a prop, unless we make it a not-prop, we will still print _some_ UIDs. I think in the short term it's better to leave the system of uid printing as-is, and only remove it once we believe we can fully remove it (ie not even pretend it's a prop, attach it to the parsed JSXElement in some other way)